Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -19,9 +12,13 @@ class WeatherForCityViewModelImpl: WeatherForCityViewModel {
     var weathers: [WeatherForCity] = []
 
     func getWeatherForCity(cityName: String) async {
-        guard let requestURL = Bundle.main.object(forInfoDictionaryKey: "WeatherRequestURL") as? String,
-              let url = URL(string: requestURL + "?q=\(cityName)" +
-                  "&appid=\(EidolonKeys().openWeatherApiKey)&units=metric")
+        let city = cityName.removeDiacratics()
+        let isContained = weathers.contains {
```

Comment:
Could you explained? What does `isContained` mean

Reply:
It checks we have already added this city  in weathers array.